bin/configs/typescript-angular-v11-oneOf.yaml

```diff
@@ -0,0 +1,5 @@
+generatorName: typescript-angular
+outputDir: samples/client/petstore/typescript-angular-v11-oneOf/builds/default
+inputSpec: modules/openapi-generator/src/test/resources/3_0/oneOfArrayMapImport.yaml
```

Comment:
Why not add this schema info to an existing typescript-angular-v11 sample?

Reply:
Trying to reuse the spec made other languages fail in unexpected fashion because of CI setup, as mentioned by Hippo earlier.